overlord/snapstate/snapstate_test.go

```diff
@@ -147,6 +146,20 @@ func (s *snapmgrTestSuite) TearDownTest(c *C) {
 	snapstate.CanAutoRefresh = nil
 }
 
+func (s *snapmgrTestSuite) TestStore(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	sto := &store.Store{}
+	snapstate.ReplaceStore(s.state, sto)
+	store1 := snapstate.Store(s.state)
+	c.Check(store1, Equals, sto)
+
+	// cached
+	store2 := snapstate.Store(s.state)
+	c.Check(store2, Equals, sto)
```

Comment:
is it being equal what we want, or is it a detail? I would've expected `DeepEquals`

Reply:
no, Equals is right here,   the cache is in-memory, so the exact objects are kept